xml/System.Reflection/MethodImportAttributes.xml

```diff
@@ -162,7 +170,7 @@
       </ReturnValue>
       <MemberValue>1280</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Specifies that the calling convention is FastCall.</summary>
```

Comment:
This is propbably because there's not enough bits to make all constants single-bit. The calling convention mask is three bits (11100000000), there are three single-bit constants:

```
CallingConventionThisCall 10000000000
CallingConventionCDecl    01000000000
CallingConventionWinApi   00100000000
```

Other two constants are combinations of these values. So this enum goes against the usual flags enum conventions, and checking like `attributes.HasFlag(MethodImportAttributes.CallingConventionWinApi)` will be bugged. I'll add a note how to properly check values.